windows_containers/configuring-byoh-windows-instance.adoc

```diff
@@ -0,0 +1,65 @@
+[id="configuring-byoh-windows-instance"]
+= Configuring Bring-Your-Own-Host (BYOH) Windows instances
+
+-NEED AN INTRO-
+
+== Prerequisites
+
+Any Windows instances that are to be attached to the cluster as a node must fulfill the following requirements:
+
+* The Docker container runtime must be installed on the instance.
+* The instance must be on the same network as the Linux worker nodes in the cluster.
+* Port 22 must be open and running an SSH server.
+* Port 10250 must be open for log collection.
+* An administrator user is present with the private key used in the secret set as an authorized SSH key.
+* The hostname of the instance must follow the link:https://datatracker.ietf.org/doc/html/rfc1123[RFC 1123] DNS label standards, which include:
+** Contain only lowercase alphanumeric characters or '-'.
+** Start with an alphanumeric character.
+** End with an alphanumeric character.
+
+== Procedure
+. Create a ConfigMap named `windows-instances` in the WMCO namespace that describes the Windows instances to be added.
++
+[NOTE]
+====
+Each entry in the data section of the ConfigMap should be formatted with the address as the key and a value with the
+format of `username=<username>`.
+====
++
+.Example ConfigMap
+[source,yaml]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: windows-instances
+  namespace: openshift-windows-machine-config-operator
+data:
+  10.1.42.1: |- <1>
+    username=Administrator <2>
+  instance.dns.com: |-
+    username=core
+----
+<1> An address to SSH into the instance with. This can be a DNS name or an IPv4 address.
+<1> The name of the administrator user created in the prerequisites.
+
+== Removing BYOH Windows instances
```

Comment:
This should also be a separate module that you can reference with an include statement here in the assembly file.